rmgpy/data/test_data/thermo/groups/adsorptionPt.py

```diff
@@ -2,8 +2,8 @@
 # encoding: utf-8
 
 name = "Surface Adsorption Corrections"
-shortDesc = u""
```

Comment:
We should remove them from RMG-database, but it's not crucial. I removed them from the testing databases because it was straightforward to do.

Ideally, we would machine write the entire database, like we planned to do a while back. Another consideration is whether or not we want to change the database syntax.